lib/constructs/interface.js

```diff
@@ -317,11 +446,11 @@ class Interface {
         return `${value} !== ${unsupportedValue}`;
       }
       return `${O}[impl][utils.supportsPropertyName](${P})`;
-    }
+    };
 
     // "named property visibility algorithm"
     // If `supports` is true then skip the supportsPropertyName check.
-    function namedPropertyVisible(P, O, supports = false) {
+    const namedPropertyVisible = (P, O, supports = false) => { // eslint-disable-line func-style
```

Comment:
Again please stick with the normal function style; if you're disabling a line rule that's a sign something has gone wrong style-wise.